docs/compilation/composing_functions_with_modules.md

```diff
@@ -43,6 +43,12 @@ x_dec = CounterFhe.inc.decrypt(x_enc)
 assert x_dec == 15
 ```
 
+Note that, if needed (for benchmarking purpose for instance), the keyset can be generated beforehand by calling `keygen()` method on the compiled module:
```

Comment:
"for benchmarking purpose for instance": it's also to use modules in real life! 

I would maybe just say:

The keyset can be generated beforehand by calling `keygen()` method on the compiled module: